src/fsharp/FSharp.Core/string.fsi

```diff
@@ -98,26 +144,115 @@ namespace Microsoft.FSharp.Core
         ///
         /// <returns>The constructed string.</returns>
         /// <exception cref="T:System.ArgumentException">Thrown when <c>count</c> is negative.</exception>
+        /// 
+        /// <example id="init-example-1"> Enumerate digits ASCII codes
+        /// <code lang="fsharp">
+        /// String.init 10 (fun i -> int '0' + i |> sprintf "%d ")  // evaluates "48 49 50 51 52 53 54 55 56 57 "
```

Comment:
These lines are getting too long I think (102 characters), suggest a maximum of 80 with a preference for 60
```fsharp
        /// <code lang="fsharp">
        /// String.init 10 (fun i -> int '0' + i |> sprintf "%d ")
        /// </code>
        /// evaluates to
        /// <code>
        /// "48 49 50 51 52 53 54 55 56 57 "
        /// </code>
```

across all the samples.